pages/about.html

```diff
@@ -1,12 +1,41 @@
-<!DOCTYPE html>
+<!DOCTYPE html> 
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
+  <meta charset="UTF-8"> 
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>About</title>
+  <link href="../styles/style.css" rel="stylesheet"/>
 </head>
 <body>
-  
+  <header>
+    <h1 class="page-title">About Alie </h1>
+    <nav> 
+      <ul> 
+        <li><a class="link" href="html_index.asp">Index</a></li>
+        <li><a class="link" href="html_about.asp">About</a></li>
```

Comment:
Similar to the `page-title` class, the `link` class isn't necessary here since you can just style the `a` element instead (and it doesn't add any special meaning):

```suggestion
        <li><a href="html_index.asp">Index</a></li>
        <li><a href="html_about.asp">About</a></li>
```